src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -87,11 +73,11 @@ public DriveSubsystem() {
     resetEncoders();
     setupEncoderConversions();
 
-    // Initialize the tracking of the robot's position on the field
-    // m_odometry = new DifferentialDriveOdometry(new Rotation2d(getHeading()));
+    // Configure a deadband of 0.1 for arcade/tank drive
+    m_drive.setDeadband(0.1);
```

Comment:
We need a deadband on the XBoxController side, not for any other uses of the Drive Subsystem.

Reply:
I guess the variable naming is a bit ambiguous here, but in this case the deadband is being applied to the `DifferentialDrive` instance within `DriveSubsystem`, which is responsible for arcade/tank drive using the sticks. Since that's the only place where the sticks are used, I figured we could move it there instead of deadbanding them individually.